ci/long_running_tests/workloads/serve_scalability.py

```diff
@@ -0,0 +1,67 @@
+# A multi-node scalability test. We put an http proxy on the head node and spin
+# up e.g. 20 nodes and put a bunch of replicas on the cluster, and run a stress
+# test.
+
+import logging
+import time
+import subprocess
+import requests
+
+import ray
+from ray import serve
+from ray.serve import BackendConfig
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+ray.init(address="auto")
+client = serve.start()
+
+# These numbers need to correspond with the autoscaler config file.
+# The number of remote nodes in the autoscaler should upper bound
+# these because sometimes nodes fail to update.
+num_remote_nodes = 4
+head_node_cpus = 2
+num_remote_cpus = num_remote_nodes * head_node_cpus
+
+# Wait until the expected number of nodes have joined the cluster.
+while True:
+    num_nodes = len(ray.nodes())
+    logger.info("Waiting for nodes {}/{}".format(num_nodes,
+                                                 num_remote_nodes + 1))
+    if num_nodes >= num_remote_nodes + 1:
+        break
+    time.sleep(5)
+logger.info("Nodes have all joined. There are %s resources.",
+            ray.cluster_resources())
+
+
+def hey(_):
+    time.sleep(0.01)  # Sleep for 10ms
+    return "hey"
```

Comment:
very minor but we should return `b"hey"` here because it skips the ray serialization codepath

Reply:
same for all of these tests